src/ledger/insert_shred.zig

```diff
@@ -89,6 +91,7 @@ pub const ShredInserter = struct {
     ///     - The slot is not currently full
     ///     It means there's an alternate version of this slot. See
     ///     `check_insert_data_shred` for more details.
+    ///    // QUESTION: How are these various family corrolated with one another together?
```

Comment:
data shreds store the data, coding shreds are used to recover missing data shreds 

https://github.com/solana-foundation/specs/blob/main/p2p/shred.md